### ISSUE TITLE
If lost connection miner not resume 

### ISSUE THREAD

**Issue by chatre7** [Optiminer/OptiminerZcash#36]
If lost connection miner not resume

**Comment by tytanick** [comment]
Yep, i just got the same.
I lost internet connection (only internet, whiel the link was online and dhcp was working),
Could not resume mining :)
Thx

**Comment by Optiminer** [comment]
Will be fixed in next release.

**Comment by stallman** [comment]
I have similar problem.
```
2016-11-16 10:18:57,612 ERROR [default] Failed to send message!
2016-11-16 10:18:57,612 ERROR [stratum] IO/Error communicating to eu1-zcash.flypool.org
2016-11-16 10:57:53,397 FATAL [default] CRASH HANDLED; Application has crashed due to [SIGINT] signal
2016-11-16 10:57:53,397 WARN  [default] Aborting application. Reason: Fatal log at [../logging/easylogging++.h:5627]
```

**Comment by jstefanop** [comment]
@Optiminer will this be fixed anytime soon...miner is currently useless if it cant reconnect, since there are mutiple connection drops per day pool side and user side.

**Comment by tytanick** [comment]
In my simplemining.net OS i am using script which restarts miner if such
message appears.

17 lis 2016 02:10 "jstefanop" <notifications@github.com> napisał(a):

**Comment by Optiminer** [comment]
Should be fixed in v0.3.4.
